pkg/services/preference/prefimpl/pref.go

```diff
@@ -247,3 +271,68 @@ func (s *Service) GetDefaults() *pref.Preference {
 func (s *Service) DeleteByUser(ctx context.Context, userID int64) error {
 	return s.store.DeleteByUser(ctx, userID)
 }
+
+func (s *Service) saveEntitty(ctx context.Context, p *pref.Preference) error {
```

Comment:
```suggestion
func (s *Service) saveEntity(ctx context.Context, p *pref.Preference) error {
```
excellent / awful typo 😁

Reply:
OMG